create_infotexts.py

```diff
@@ -406,7 +406,8 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
             place_matches = []
             for place in places_mapping:
                 if place in desc_string:
-                    place_matches.append(place)
+                    if not re.search(r"Nicosiavägen", desc_string, re.IGNORECASE):
```

Comment:
Could you not do this using `if not 'nicosiavägen' in desc_string.lower()`? That way we avoid having to use `re`